docs/working-with-projects/working-with-images/deleting-artifact.md

```diff
@@ -37,13 +37,13 @@ An example of an OCI image index
 }
 ```
 
-**Deleting Artifact**:
+**Deleting an Artifact**
 
-When an artifact is not referenced by any OCI index, you can delete the artifact freely which will delete its manifest and all associated tags. 
+When an artifact is not referenced by any OCI index, you can delete the artifact freely which also deletes its manifest and all associated tags. 
 
-When an artifact is referenced by an OCI index, you cannot delete it.  In order to delete this artifact, you must first delete all OCI indexes referencing this artifact first, remembering that an artifact can be referenced by multiple parents artifacts pushed onto Harbor by different users.  So when deleting an OCI index holding 9 children artifacts not referenced by any other index and 1 child artifact referenced by another index, only 9 out of 10 children artifacts will be deleted.
+When an artifact is referenced by an OCI index, you cannot delete it.  To delete a referenced artifact, you must first delete all OCI indexes referencing the artifact. Remember that an artifact can be referenced by multiple parent artifacts pushed onto Harbor by different users.  So when you delete an OCI index holding 9 child artifacts that are not referenced by any other index and 1 child artifact referenced by another index, only 9 out of 10 child artifacts are deleted.
```

Comment:
'remember too' is too colloquial, change to ' An artifact can also be referenced...'

Reply:
'So when you delete ' -> ' When you delete ' ?